cpp/src/pypowsybl-api.h

```diff
@@ -364,9 +364,11 @@ typedef enum {
 } DynamicMappingType;
 
 typedef enum {
-    ONE = 0,
+    UNDEFINED = -1,
```

Comment:
@EtienneLt  I had a small issue here because NONE is an enum value already used and graalvm seems to not like that somehow. So NONE in python -> UNDEFINED on cpp/java .

Reply:
ok works for me